CondFormats/GEMObjects/src/classes.h

```diff
@@ -0,0 +1,8 @@
+#include "CondFormats/GEMObjects/interface/GEMEMap.h"
```

Comment:
is this file needed?

Reply:
yes, i think so. Does not compile without this file

Reply:
From CondObjectsTutorial:
"We also need a src/headers.h file, which holds include statements to all our serializable classes. This file is read by the serialization system to find the serializable classes and generate serialization code for them."